src/Apia/Options.hs

```diff
@@ -46,10 +45,12 @@ module Apia.Options
            , optWithIleanCoP
            , optWithMetis
            , optWithSPASS
+           , optWithtptp2X
```

Comment:
~~Please create a different PR for adding this missing option.~~
`tptp2X` is not currently used. Are you working with an update version of the repository?